libraries/azurerm_ad_user.rb

```diff
@@ -52,6 +52,6 @@ def guest?
   end
 
   def to_s
-    "Azure Active Directory Username: '#{displayName}' with email '#{mail}'"
+    "Azure Active Directory Username: '#{displayName}' with objectId '#{objectId}'"
```

Comment:
Is the User Id a good thing to put here to identify the user? An email address would be easier to look up in your system wouldn't it?

Reply:
Email would be nicer - but it is a nullable field and so leads to unreliable output